README.md

```diff
@@ -24,6 +24,21 @@ colorscheme):
 import:
  - ~/.config/alacritty/themes/themes/{theme}.yaml
 ```
+### For Windows users
+Clone the repository, or download the theme of your choice:
+
+```sh
+mkdir %USERPROFILE%\AppData\Roaming\alacritty\themes
+git clone https://github.com/alacritty/alacritty-theme %USERPROFILE%\AppData\Roaming\alacritty\themes
+```
+
+Please replace `{username}` with the actual username and `{theme}` with the desired theme name
+
+```yaml
+#Include only absolute path
+import:
+  - C:\Users\{username}\AppData\Roaming\alacritty\themes\themes\{theme}.yaml
+```
```

Comment:
The import instructions work just fine on Windows with a proper unix shell, so I don't see a reason to have shitty Windows script instructions that nobody would ever want to use anyway.

The paths in Rust are platform-agnostic, so if you put it in the right location, it will work on both platforms.

Reply:
I thought that not all Windows users may have a Unix shell or be familiar with it. Therefore, providing platform-specific instructions can be beneficial to cater to a wider audience. However, as you rightly pointed out, since not many users are likely to utilize the Windows script, I have removed it from the Readme.md file.